source/includes/steps-configure-windows-service-for-mongodb.yaml

```diff
@@ -46,8 +46,39 @@ inherit:
   file: steps-create-manually-windows-service-for-mongodb.yaml
   ref: start-mongodb-service
 ---
-title: Stop or remove the MongoDB service as needed.
 stepnum: 6
+source:
+  file: steps-run-mongodb-on-linux.yaml
+  ref: verify
+pre: |
+  Verify that MongoDB has started successfully by checking the 
+  log file at ``c:\data\log\mongod.log`` for the following line:
+post: |
```

Comment:
what line?

Reply:
I think this pulls in the `action` from the inherited file, so the line ends up being the `initAndListen` log file. Staged here: https://docs-mongodbcom-staging.corp.mongodb.com/rkumar/DOCS-11126/tutorial/install-mongodb-on-windows.html#verify-that-mongodb-has-started-successfully